tests/parser-test-files/package.txt

```diff
@@ -36,7 +36,7 @@ error: Malformed package declaration
   --> test:1:8
    |
  1 |  (package)
-   |          ^ missing package name
```

Comment:
Personally I kind of prefer the pointer to be under the `)` because it makes it clear that it's pointing to the next element.

Reply:
In this case the note constructor is given the zero-indexed span (8 . 8) which ought to indicate the gap between `e` and `)`, but the condition printer is emitting 7 spaces instead of 8. I believe there are a few dozen other cases in the text-formatted error test cases. I'll open a separate cleanup issue for it.

Reply:
This issue filed as #1248